clarity-ext-scripts/clarity_ext_scripts/covid/fetch_biobank_barcodes.py

```diff
@@ -0,0 +1,137 @@
+import re
+from clarity_ext.service.file_service import Csv
+from clarity_ext.domain.validation import UsageError
+from clarity_ext.utils import single
+
+BIOBANK_FILE_HEADER = ['well', 'biobank_barcode', 'something_else', 'plate_barcode']
+
+
+class FetchBiobankBarcodes(object):
+    """
+    This is intended to be part of create samples script. By injecting context,
+    it might be tested in isolation by creating a dummy script.
+    """
+
+    def __init__(self, context):
+        self.context = context
+        self.barcode_by_sample_code = None
+
+    def execute(self):
+        """
+        Just for testing, validate() and map_barcodes() are called separately from
+        different scripts
+        """
+        self.validate()
+        self.barcode_by_sample_code =\
+            self.biobank_barcode_by_sample_referal_code()
+        self._print(self.barcode_by_sample_code)
```

Comment:
Actually, it's only for test. validate() is called from 'validate_sample_creation_list' and 'biobank_barcode_by_sample_referal_code()' is called from 'import_samples'. But it would be super hard to call these scripts during development.